CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ See https://6.dev-docs.plone.org/volto/upgrade-guide/index.html for more informa
 
 - Japanese translation updated @terapyon
 - Improve the `AlignWidget`, add `narrow` fix default support @sneridagh
+- Standardize the existence of core `packages` folder. You can define a number of core add-ons present in this folder and they get configured automatically @sneridagh
```

Comment:
@sneridagh what does "configured automatically" mean? I see below that you have to put packagesFolderAddons in package.json, which doesn't look automatic.

And I'm not sure I understand the point of making it configurable in volto's package.json instead of just hardcoding the list of packages to load.

Reply:
Yeah, I had this doubt as well... I had a quick talk to @tiberiuichim and it seemed to us that could be interesting at some point that the list is more "accessible" in terms to be able to configure it, if required.

Background: I want to start developing quanta as a core package, located in the `packages` folder. Same for other potential experimental features that could be implemented as add-ons, but located also in core. These add-ons could be published as separated add-ons so they could be installed in projects separately.

Reply:
Using this approach, separated from core, is also less prone to have conflicts and you can locate it in long life branches.

Reply:
I will reword the Changelog.

Reply:
@sneridagh It makes sense to me to have the ability to put optional addons in the volto repository. I'm just confused because you're describing something that is optional per package, but the PR description and the changelog both say that all packages will be loaded automatically. Maybe they are just out of date?

Reply:
@davisagli 

For context and "background": our [packages](https://github.com/plone/volto/tree/master/packages) folder right now has a bunch of things that are not Volto addons, so we can't really use that location for an automatic "all these are volto addons". volto-slate was the first thing that was added there, and to make it work, we include a special case in the addons-registry so that the webpack config to point to that location. Victor's PR fixes the hack by making it a pattern. So it's really an internal setting for Volto itself, nothing that would concern projects, so it doesn't have to be pretty.